--- soqt/SoQt.pc.cmake.in.orig	2019-12-24 01:35:10.000000000 +0100
+++ soqt/SoQt.pc.cmake.in	2020-12-01 21:29:56.792591735 +0100
@@ -1,13 +1,13 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-bindir=${exec_prefix}/bin
-libdir=${exec_prefix}/lib
-includedir=${prefix}/include
-datarootdir=${prefix}/share
-datadir=${datarootdir}
-docdir=${datarootdir}/doc/@PROJECT_NAME_LOWER@
-infodir=${datarootdir}/info
-mandir=${datarootdir}/man
+bindir=@CMAKE_INSTALL_FULL_BINDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+datarootdir=@CMAKE_INSTALL_FULL_DATAROOTDIR@
+datadir=@CMAKE_INSTALL_FULL_DATADIR@
+docdir=@CMAKE_INSTALL_FULL_DOCDIR@
+infodir=@CMAKE_INSTALL_FULL_INFODIR@
+mandir=@CMAKE_INSTALL_FULL_MANDIR@
 htmldir=${docdir}/html
 
 Name: @PROJECT_NAME@
@@ -15,7 +15,7 @@
 Version: @PROJECT_VERSION@
 Requires: Coin, @PACKAGE_ADDITIONAL_REQUIREMENTS@
 Conflicts:
-Libs: -L${libdir} @SOQT_EXTRA_LDFLAGS@ @SOQT_EXTRA_LIBS@
+Libs: -L${libdir} -lSoQt @SOQT_EXTRA_LDFLAGS@ @SOQT_EXTRA_LIBS@
 Cflags: -I${includedir} @SOQT_EXTRA_CFLAGS@ @SOQT_EXTRA_CPPFLAGS@
 
 frameworkdir=@frameworkdir@
